api/src/setupPlugins.ts

```diff
@@ -0,0 +1,14 @@
+import { setup } from 'powerhouse-switchboard-plugin-demo';
```

Comment:
Why does one plugin requires calling a setup when the other does not?

Reply:
- one plugin has to have some crud defined already, another one does not.
- one plugin relies on the api_origin env var, another one does not.

Reply:
So the suggestion is to make this interface universal: all plugins have to return `setup` function, which will receive the same context object with env vars etc. If a plugin doesn't need the context, it just doesn't use it.

Reply:
see the current version